sdk/src/logs/batch_log_processor.cc

```diff
@@ -227,6 +235,7 @@ void BatchLogProcessor::Export()
           nostd::span<std::unique_ptr<Recordable>>(records_arr.data(), records_arr.size()));
       NotifyCompletion(notify_force_flush, synchronization_data_);
     }
+#  ifdef ENABLE_ASYNC_EXPORT
```

Comment:
Just like in `SimpleSpanProcessor`, `if (is_export_async_ == false)` should only be checked when `ENABLE_ASYNC_EXPORT` is defined, or we should always call sync Export.

Reply:
Done